nototools/generate_website_2_data.py

```diff
@@ -554,7 +554,7 @@ def get_css_generic_family(family):
 
 CSS_WEIGHT_MAPPING = {
     'Thin': 250,
```

Comment:
This PR is about web-site update. Sorry if I'm slow. 
Do you use these values in CSS?  In that case, 100 for Thin and 200 for ExtraLight should just work fine and those values are what to use in CSS. Only inside a font file, we don't use anything lower than 250 for OS/2.fsWeight to make GDI happy.